demo/src/screens/componentScreens/SliderScreen.tsx

```diff
@@ -124,7 +135,28 @@ export default class SliderScreen extends Component<SliderScreenProps, SliderScr
           />
 
           <Text $textDefault text70BO marginV-15>
-            Range slider
+            Range Slider
+          </Text>
+          <View row spread style={this.getReverseStyle()}>
+            <Text bodySmall $textNeutral>
+              min. {sliderMinValue}%
+            </Text>
+            <Text bodySmall $textNeutral>
+              max. {sliderMaxValue}%
```

Comment:
The presented values of the two range sliders are the same (even though the values are different)..

Reply:
I'll take a look. Thanks

Reply:
Done